library/general/disallowlatest/template.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sdisallowlatest
+  annotations:
+    description: Requires container images to contain an image tag different that "latest".
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sDisallowLatest
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8sdisallowlatest
+
+        violation[{"msg": msg}] {
+          container := input.review.object.spec.containers[_]
+          latest := [re_match(":latest", container.image)]
+          all(latest)
+          msg := sprintf("container <%v> uses a latest image tag <%v>", [container.name, container.image])
+        }
+
+        violation[{"msg": msg}] {
+          container := input.review.object.spec.containers[_]
+          tag := [re_match(":", container.image)]
```

Comment:
The `contains()` builtin is probably best to use here:

https://www.openpolicyagent.org/docs/latest/policy-reference/#strings